tsconfig.json

```diff
@@ -6,7 +6,7 @@
     "esModuleInterop": true,
     "importHelpers": true,
     // See https://devblogs.microsoft.com/typescript/typescript-and-babel-7/
-    "isolatedModules": true,
+    "isolatedModules": false,
```

Comment:
Fails if we set this to true. I've read a bit about it but not 100% sure. Did it fail in earlier versions of babel v7 when being true?

Reply:
Hmm as far as I know, if we use Babel we need to activate this flag. Did you find a reference or docs that say otherwise?

Also, I'm not sure how this change relates to any of the updated dependencies. What was the error that you got? 🤔

Reply:
<img width="1648" alt="CleanShot 2020-02-11 at 09 20 21@2x" src="https://user-images.githubusercontent.com/1877073/74220387-be111600-4caf-11ea-96c1-70229e2954f6.png">

I've also not found much otherwise. However, also confusing that it works. Typechecking jest however fails without.

Reply:
See https://github.com/chalk/chalk/issues/295 and https://github.com/chalk/chalk/pull/296#issuecomment-418789492

Reply:
https://github.com/chalk/chalk/issues/382

<img width="547" alt="image" src="https://user-images.githubusercontent.com/1110551/74221639-e0f0f980-4cb2-11ea-93d8-27babdbfe781.png">

Reply:
I'm wondering if we still need the `@types/jest`. 

In any case, we need to keep `isolatedModules: true` and try to downgrade one of the packages so that we don't get the problem with the chalk dependency

Reply:
Yes, read that chalk issue too. It's a bit of a cat and mouse game. I now:

1. Downgraded ts-jest
2. Removed @types/jest (comes with jest itself)

This should work. However, it's a general issue: if any library uses language TS features babel can't compile we're doomed. This is the case somehow with the updated dependency of calk in ts-jest.